### PR TITLE
Re-ading all the fixes that were reverted, except the one that bugged

### PR DESCRIPTION
**Pull request by Sir-Chili**

### CODE REVIEW

File: randomizer/Patching/CosmeticColors.py [hunk 1]
```diff
@@ -2531,6 +2556,26 @@ def placeKrushaHead(slot):
     """Replace a kong's face with the Krusha face."""
```
Comment (AlmostSeagull) [inline randomizer/Patching/CosmeticColors.py:2556]: Forgot the colorblind mode check here, see PR #2204

**Review by AlmostSeagull — CHANGES_REQUESTED** [review]

**Comment by AlmostSeagull** [comment]
The rest looks good! I assume you did build. (no need to rebuild after implementing the change I requested)

File: randomizer/Patching/CosmeticColors.py (hunk 1 above)
Comment (Sir-Chili) [reply randomizer/Patching/CosmeticColors.py:2556]: Figures I'd forget something. Adding it back.

**Comment by Sir-Chili** [comment]
Built and tested fully this time, yes. Also ended up documenting what uses each portrait.

**Comment by AlmostSeagull** [comment]
Okay, so remember how I said you don't need to rebuild after making that change? Well, I pushed an update, so you'll have to pull dev, rebuild and commit that again, just like last time.

**Comment by Sir-Chili** [comment]
Haha, yeah, just realised that too. No worries, I'll do the whole song and dance, then.

**Pull request closed without merging**